src/selectors/prometheus/selectors.js

```diff
@@ -28,3 +28,40 @@ export const getLastUtilizationStat = response => {
   const history = getUtilizationVectorStats(response);
   return history ? history[history.length - 1] : null;
 };
+
+export const getTopConsumerVectorStats = result => {
```

Comment:
can you please move this to `prometheus/storage.js` ? it is getting overcrowded here IMO.

Reply:
ack